### DISCUSSION TITLE
EO_INSPIRE-DM_Earth observation Data Model

### DISCUSSION THREAD

**Discussion by meriamlahsaini** [INSPIRE-MIF/helpdesk#217] [General]
I am currently working with the Knowledge Center for Earth Observation and we have begun developing an Earth Observation (EO) data model. I am interested in integrating the INSPIRE Data Model (https://github.com/INSPIRE-MIF/technical-guidelines/tree/main/data ), but I am facing challenges in finding a suitable theme for this integration. Do you have any suggestions on how to approach this situation or any ideas on how to proceed?

**Comment by sMorrone** [comment]
@meriamlahsaini:  while EO is not a distinct INSPIRE theme, several themes can align depending on your data type and use case.
Below some hints:

- Orthoimagery (OI) -Directly relevant if you're working with satellite images or derived raster products. 
- Land Cover (LC) -Suitable for EO products that classify surface features (e.g., CORINE-type datasets, NDVI-based vegetation monitoring).
- Environmental Monitoring Facilities (EF) -Good fit for sensor platforms and derived observation data (e.g., pollutant monitoring, ecosystem status indicators).
- Soil (SO), Land Use (LU), or Agricultural and Aquaculture Facilities (AF)- Relevant if your EO data supports sector-specific analysis (e.g., farming practices, land use chang-es).
- Natural Risk Zones (NZ): For EO supporting disaster monitoring or hazard mapping (e.g., floods, fires).
- Oceanographic Geographical Features (OF): For EO data focused on marine/coastal areas.

In addition, you could extend the INSPIRE data themes to suit your specific needs (see https://inspire-extensions.wetransform.to/)
